package.json

```diff
@@ -46,6 +46,6 @@
   "license": "MIT",
   "dependencies": {
     "hoek": "~4.1.0",
-    "raven": "~1.1.1"
+    "raven": "~2.0.2"
```

Comment:
Can you please split this into two separate commits? One with dev dependencies and the other with production ones.

( Personally I like to keep packages grouped while updating, e.g. bump babel-* versions, test and commit, then other packages. It's easier to verify and revert if there are problems. )